configuration/configure-storage-management.md

```diff
@@ -28,10 +34,10 @@ Once finished, **click Save (6)** to save your storage management preferences.
 
 ## Related Articles
 
-* [Optimize Storage](../governance-and-automation/optimize-storage.md)
+* [Storage Management](../reporting/storage-management/storage-management.md)
 * [Configure Tenant Storage Limit Policy](../governance-and-automation/automated-workflows/tenant-storage-admin.md)
 * [Tenant Storage Security & Compliance Check](../../governance-and-automation/security-compliance-checks/tenant-storage.md)
 * [Resolve Tenant Storage Limit Task](../../point-collaborators/resolve-governance-tasks/tenant-storage-limit.md)
-* [Storage Metrics Report](../reporting/cleanup-and-health-reports.md#storage-metrics)
-* [Site Storage Metrics Report](../reporting/cleanup-and-health-reports.md#site-storage-metrics)
-* [File Storage Details Report](../reporting/cleanup-and-health-reports.md#file-storage-details)
\ No newline at end of file
+* [Storage Metrics Report](../reporting/storage-management/storage-reports.md#storage-metrics)
```

Comment:
ova promjena iz cleanup-and-health u storage-management znači da i svi articlei koji sadrže link na stare članke moraju biti updateani. našao sam issue u sljedećim article-ima
configuration\configure-storage-management.md
point-collaborators\resolve-governance-tasks\tenant-storage-limit.md
reporting\storage-management\storage-management.md